examples/movies/server/src/movies/Main.scala

```diff
@@ -63,6 +63,7 @@ class Serve extends TwitterServer with StoreKit with TreodeAdmin {
       BadRequestFilter andThen
       JsonExceptionFilter andThen
       PeersFilter ("/peers", controller) andThen
+      LoggingFilter andThen
```

Comment:
This is already in the filter chain on line 61. Doesn't adding it here make for double logging? Is there something wrong about how it was setup on the line above?